src/render.py

```diff
@@ -62,3 +69,27 @@ def print_board(board, mode=PRINT_BOARD_PLACE_MARBLE, clear_before_printing=True
     else:
         print_board_rotate(board)
 
+def print_players(players, current_player_id):
+
+    # If current player is 1, draw around his nickname a box.
+    # Instead draw this box on the second player.
+    if current_player_id == players[0]["id"]:
+        print("    ┌────────────┐")
```

Comment:
the content of the if and else should/must be factorize, you print the same thing

Reply:
Yes, I tried another version very factorized and very complicated. That's why I decided to duplicate some things.
I will suggest an half-way solution.